src/distributions.jl

```diff
@@ -27,16 +27,17 @@ function Base.rand(rng::AbstractRNG, μ::DistributionMeasure{F,S,X}) where {F, S
     return rand(rng, μ.dist)
 end
 
-export Normal
-
-
 basemeasure(d::Dists.AbstractMvNormal) = Lebesgue(ℝ)^size(d)
 
 function basemeasure(μ::Dists.Distribution{Dists.Univariate,Dists.Continuous})
     return Lebesgue(ℝ)
 end
 
+∫(::typeof(identity), ::Dists.Distribution) = 1.0
 
 logdensity(μ::Dists.Distribution, x) = Dists.logpdf(μ,x)
+Dists.logpdf(d::AbstractMeasure, x) = logdensity(d,x)
```

Comment:
should this mimic distribution's `pdf`, which is kind of broken? If we would like to represent the actual behaviour
```
function Dists.logpdf(d::AbstractMeasure, x)
    μ = basemeasure(d)
    if μ ≪ Lebesgue(...) # depending on size?
        logdensity(d,Lebesgue, x)
    elseif μ ≪ CountingMeasure(...)
          logdensity(d,CountingMeasure, x)
    else
          error("logpdf undefined, try logdensity")
    end
end
```

Reply:
Ahh, good point. I'd like to avoid the conditionals if possible, since they can make it hard to extend. But this is the idea of `representative`. There are actually some pending design decisions around that concept. This PR review won't be very visible, so maybe I'll start a new thread in Zulip about it in case anyone else is interested.

Reply:
@mschauer I added an issue on this here:
https://github.com/cscherrer/MeasureTheory.jl/issues/98

That's maybe not entirely clear - giving a clear description of something as I'm figuring it out is kind of tricky. I think what it comes down to is that we can have

```julia
function Dists.logpdf(d::AbstractMeasure, x)
    b = fix(basemeasure, d)
    return logdensity(d, b)
end
```

This traverses to a fixpoint twice (once to find the fixpoint, and again to compute the `logdensity`). I'm sure we can make this more efficient. When we do, we can add a test

```julia
d = ...
x = rand(d)
@test Dists.logpdf(d, x) == logdensity(d, fix(basemeasure, d), x)
```

What do you think?